platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/rest/ApplicationUploadEndpoint.java

```diff
@@ -159,73 +155,65 @@ public Response create(MultipartBody multipartBody, @Context UriInfo requestUriI
    * @param attachment the attachment to copy and extract.
    * @return The file of the copied attachment.
    */
+  @Nullable
```

Comment:
❓ Isn't nullable supposed to be on the parameter instead of the method?

Reply:
I can be used either way, but in our case, google style enforces annotations be declared first.

Reply:
Clarifying for @bakejeyner: the annotation can be on _in_ parameters or on the _out_ result. In this case, it indicates that the `File` returned can be null.